operations/_objects.md

```diff
@@ -199,4 +199,27 @@ Usage of this value is **deprecated**. Where possible, use the properties exposi
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
 | `Currency` | string | required |  |
-| `Value` | number | required |  |
\ No newline at end of file
+| `Value` | number | required |  |
+
+#### Resource Feature Classification
```

Comment:
FYI: this is not related to this task but is caused by adding enum anotation [in this task](https://github.com/MewsSystems/mews/pull/56995/files).